lib/rucio/tests/test_multi_vo.py

```diff
@@ -375,3 +377,61 @@ def test_account_counters_at_different_vos(self):
         session.query(models.UpdatedAccountCounter).filter_by(rse_id=tst_rse1_id, account=new_acc).delete(synchronize_session=False)
 
         session.commit()
+
+
+class TestMultiVODaemons(object):
+
+    @classmethod
+    def setUpClass(cls):
+        if config_get_bool('common', 'multi_vo', raise_exception=False, default=False):
+            cls.vo = {'vo': 'tst'}
+            cls.new_vo = {'vo': 'new'}
+            if not vo_exists(**cls.new_vo):
+                add_vo(description='Test', email='rucio@email.com', **cls.new_vo)
+        else:
+            LOG.warning('multi_vo mode is not enabled. Running multi_vo tests in single_vo mode will result in failures.')
+            cls.vo = {}
+            cls.new_vo = {}
+
+    def test_automatix(self):
+        """ MULTI VO (DAEMON): Test that automatix runs on a single VO """
+        scope_client = ScopeClient()
+        scope_uuid = str(generate_uuid()).lower()[:16]
+        shr_scope = 'shr_%s' % scope_uuid
+        scope_client.add_scope('root', shr_scope)
+        add_scope(shr_scope, 'root', 'root', **self.new_vo)
+
+        rse_client = RSEClient()
+        rse_str = ''.join(choice(ascii_uppercase) for x in range(10))
+        shr_rse = 'SHR_%s' % rse_str
+        mock_protocol = {'scheme': 'MOCK',
+                         'hostname': 'localhost',
+                         'port': 123,
+                         'prefix': '/test/automatix',
+                         'impl': 'rucio.rse.protocols.mock.Default',
+                         'domains': {
+                             'lan': {'read': 1,
+                                     'write': 1,
+                                     'delete': 1},
+                             'wan': {'read': 1,
+                                     'write': 1,
+                                     'delete': 1}}}
+        rse_client.add_rse(shr_rse)
+        rse_client.add_rse_attribute(rse=shr_rse, key='verify_checksum', value=False)
+        rse_client.add_protocol(shr_rse, mock_protocol)
+        new_id = add_rse(shr_rse, 'root', **self.new_vo)
+        add_rse_attribute(rse=shr_rse, key='verify_checksum', value=False, issuer='root', **self.new_vo)
+        add_protocol(rse_id=new_id, parameter=mock_protocol)  # Use the core to add protocol in order to bypass the cache
```

Comment:
We should update this line once #56 is merged, to remove the comment and maybe use the API